src/command/subgraph/push.rs

```diff
@@ -32,6 +32,12 @@ pub struct Push {
     #[structopt(long)]
     #[serde(skip_serializing)]
     service_name: String,
+
+    /// Url of a running service that a gateway can route operations to
+    /// (often a deployed service). May be left empty ("") or a placeholder url
+    /// if not running a gateway in managed federation mode
+    #[structopt(long)]
+    routing_url: String,
```

Comment:
We should probably parse this in as a `Url` instead of a `String`.

Reply:
We could do this, but we'd have to make the flag optional in that case, so we have the ability to omit it.